src/bundle_info.c

```diff
@@ -0,0 +1,553 @@
+/*
+ *   Software Updater - client side
+ *
+ *      Copyright © 2012-2019 Intel Corporation.
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation, version 2 or later of the License.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#define _GNU_SOURCE
+#include <getopt.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include "swupd.h"
+
+#define FLAG_INCLUDES 2000
+#define FLAG_FILES 2001
+
+static bool cmdline_option_includes = false;
+static bool cmdline_option_files = false;
+static int cmdline_option_version = 0;
+
+static char *bundle;
+
+struct bundle_info {
+	struct manifest *manifest;
+	struct list *direct_includes;
+	struct list *indirect_includes;
+	struct list *includes_manifests;
+	struct list *files;
+	bool is_installed;
+	bool is_experimental;
+	int version_latest_available;
+	long size_total;
+	long size_required;
+};
+
+static void print_help(void)
+{
+	print("Usage:\n");
+	print("   swupd bundle-info [OPTIONS...] BUNDLE\n\n");
+
+	global_print_help();
+
+	print("Options:\n");
+	print("   -V, --version=V  Show the bundle info for the specified version V, also accepts 'latest' and 'current' (default)\n");
+	print("   --includes       Show all bundles included by this bundle\n");
```

Comment:
not sure if that is the way to go, at some point we were always adding short options for every flag but some times it doesn't make sense so I decided to hold on those until we think about it. We can always add those later. And actually I think that even if we decide to add short options for these flags we should wait until more of the features of this command are implemented because a short option we pick now may end up making more sense for a new flag. If for example I add the short option `-f` to `--files` now, and at some point we add another flag, let's say `--force`, we would not be able to use `-f` anymore and maybe it makes more sense for --force than for --files... All I'm saying is that we should wait and see how the command looks like when is a bit more complete before deciding in short options.
I added the short option `-V` to `--version` already just because that flag is already being used in multiple commands, so we need to stay consistent with that.